### PR TITLE
Add changes related to api state change command

### PR DESCRIPTION
**Pull request by npamudika**
## Purpose
Fixes related to https://github.com/wso2/product-apim-tooling/issues/290

### CODE REVIEW

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 1]
```diff
@@ -594,6 +594,53 @@ For more information, see [Download and Initialize the CTL Tool](#download-and-i
             DefaultApplication Application deleted successfully!
             ``` 
 
+## Change status of an API in an environment
+Follow the instructions below to change the status of an API in an environment using CTL:
```
Comment (malinthaprasan) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:598]: shall we have a note that we need apictl 3.1.1 (or later) version is required?

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 2]
```diff
@@ -594,6 +594,53 @@ For more information, see [Download and Initialize the CTL Tool](#download-and-i
...
+            ```go
+            apictl change-status api --action Publish --name PizzaShackAPI --version 1.0.0 --environment production --provider Alice --insecure
+            ```  
+
+    -   **Response**
+
+        ```go
+        PizzaShackAPI API state changed successfully!
+        ```
+
```
Comment (malinthaprasan) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:643]: Shall we also add the list of supported "Actions"?

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md (hunk 1 above)
Comment (npamudika) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:602]: added

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md (hunk 2 above)
Comment (npamudika) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:651]: done

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 3]
```diff
@@ -471,8 +471,12 @@ For more information, see [Download and Initialize the CTL Tool](#download-and-i
 ## Delete an API/API Product/Application in an environment
 Follow the instructions below to delete an API/API Product/Application in an environment using CTL:
 
-1.  Make sure that the WSO2 API Manager 3.1.0 version is started and that the 3.1.0 version of APTCTL is running.   
+1.  Make sure that the WSO2 API Manager 3.1.0 version is started and that the 3.1.1 version of APTCTL is running.   
 For more information, see [Download and Initialize the CTL Tool](#download-and-initialize-the-ctl-tool).
+
+    !!! Note
+        Note that you need to have the latest patch release of APICTL 3.1.0 version (3.1.1) to use this feature.
```
Comment (malinthaprasan) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:478]:
```suggestion
        Note that you need to have the APICTL version 3.1.1 or higher to use this feature.
```

**Pull request merged**